iocore/net/SSLNetVConnection.cc

```diff
@@ -1812,6 +1813,31 @@ SSLNetVConnection::populate(Connection &con, Continuation *c, void *arg)
   return EVENT_DONE;
 }
 
+void
+SSLNetVConnection::increment_ssl_version_metric(int version) const
+{
+  switch (version) {
+  case SSL3_VERSION:
+    SSL_INCREMENT_DYN_STAT(ssl_total_sslv3);
+    break;
+  case TLS1_VERSION:
+    SSL_INCREMENT_DYN_STAT(ssl_total_tlsv1);
+    break;
+  case TLS1_1_VERSION:
+    SSL_INCREMENT_DYN_STAT(ssl_total_tlsv11);
+    break;
+  case TLS1_2_VERSION:
+    SSL_INCREMENT_DYN_STAT(ssl_total_tlsv12);
+    break;
+  case TLS1_3_VERSION:
```

Comment:
Probably need to ifdef TLS1_3_VERSION since this constant won't be available if you are building against 1.1.0 or earlier.  Probably causing the build errors.

Reply:
Ah, you are right. Fixed.